src/global.d.ts

```diff
@@ -0,0 +1,7 @@
+interface MovieFromServer {
```

Comment:
rename file to types.

```suggestion
export interface Movie {
```